cdk/dcm_floodlight/transform.js

```diff
@@ -101,7 +135,10 @@ function trackPostMapper(input, mappedPayload, rudderContext) {
     throw new CustomError("[DCM Floodlight]:: Conversion event not found", 400);
   }
 
-  customFloodlightVariable = transformCustomVariable(customFloodlightVariable);
+  customFloodlightVariable = transformCustomVariable(
```

Comment:
We are considering the properties from the message only so apart from sending the message we can send the properties only. we can provide a check for if properties exist or not if CDK is not doing it in the first place.